### PR TITLE
[DOCS] Add parameter examples to EQL search tutorial

### PR DESCRIPTION
**Pull request by jrodewig**
Makes the following updates to the EQL search tutorial:

* Adds an API response to the basic tutorial
* Adds an example using the `event_type_field` parm
* Adds an example using the `timestamp_field`parm
* Adds an example using the `query` parm
* Updates example dataset to support more EQL query variety

### CODE REVIEW

File: docs/reference/eql/search.asciidoc [hunk 1]
```diff
@@ -40,8 +41,120 @@ GET sec_logs/_eql/search
...
+  "timestamp_field": "file.accessed",
+  "event_type_field": "event.category",
+  "rule": """
+    file where (file.size > 1 and file.type == "file")
+  """
+}
+----
+
+[discrete]
+[[eql-search-filter-query-dsl]]
```
Comment (astefan) [inline docs/reference/eql/search.asciidoc]: fyi https://github.com/elastic/elasticsearch/pull/52971

**Review by astefan — APPROVED** [review]
LGTM

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Comment by jrodewig** [comment]
@elasticmachine update branch

File: docs/reference/eql/search.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/eql/search.asciidoc]: thanks. Addressed with e195cee22e0a13ce6d9d224da8cf30e4e3486dc7.

**Pull request merged**